lib/emojiKeywords.json

```
@@ -65,5 +65,7 @@
   "rain": "rain_cloud",
   "linux": "linux",
   "hacktober": "hacktoberfest",
-  "hacktoberfest": "hacktoberfest"
+  "hacktoberfest": "hacktoberfest",
+  "exams": "books",
+  "studying": "orange_book"
```

One of the driving ideas of the auto-tagging is that since there are so many emojis, we want to coalesce usage into a smaller set so the individual topic pages are more full of relevant content, instead of spread-out. Maybe just use `books` here too?
```suggestion
  "exam": "books",
  "studying": "books",
  "studied": "books",
  "study": "books"
```

---

Oh cool!